spec/index.html

```diff
@@ -160,9 +155,9 @@
       </section>
       <section id="inclusion">
         <h2>Criteria for Inclusion</h2>
-        <p>The first version of this specification is not intended to be complete. It is currently impossible to make some web features accessible, so the primary goal is to resolve immediate needs quickly for existing, inaccessible web interfaces. The specification editors are purposefully deferring many useful ideas in order to maintain a realistic timeline for highest priority features.</p>
+        <p>This specification is not intended to solve all accessibility problems on the web. It is currently impossible to make some web features accessible, so the primary goal is to resolve immediate needs quickly for existing, inaccessible web interfaces. The specification editors are purposefully deferring many useful ideas in order to maintain a realistic timeline for highest priority features.</p>
```

Comment:
Sorry, I should have clarified, but I deliberately did not make any changes
to phases 2 - 4 in this patch. I just moved them.